scrapper.py

```diff
@@ -25,39 +33,94 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        for article_link in article_bs.find_all("a", class_="article__download button-icon"):
+            self.urls.append(''.join(["https://journals.kantiana.ru/", article_link.get("href")]))
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+
+        headers = {
```

Comment:
could you suggest a more configurable place for headers? especially given they will be reused several times across this particular module?

Reply:
Yes. It's possible to move to to innit.